mutator/cosmos/comparison.go

```diff
@@ -12,8 +12,8 @@ func init() {
 }
 
 var comparisonMutations = map[string]string{
-	"GT": "LTE",
-	"LT": "GTE",
+	"GT":  "LTE",
+	"LT":  "GTE",
 	"GTE": "LT",
 	"LTE": "GT",
```

Comment:
Can we either in this PR or a followup add logic to also test GT -> GTE & LT -> LTE.

(Need to make the map value field a slice I assume, and produce multiple mutations)

Reply:
Added here! https://github.com/osmosis-labs/go-mutesting/pull/9/commits/e6c5dd0adea030bf634d712c918b2cd3c16b130e